pkg/rpc/server.go

```diff
@@ -304,6 +300,28 @@ func (s *Server) getrawtransaction(reqParams Params) (interface{}, error) {
 	return results, resultsErr
 }
 
+// getaccountstate returns account state either in short or full (unspents included) form.
+func (s *Server) getaccountstate(reqParams Params, unspents bool) (interface{}, error) {
```

Comment:
why not camel case?

Reply:
I guess because of this:
```
func (s *Server) getrawtransaction(reqParams Params) (interface{}, error) {
func (s *Server) invokescript(reqParams Params) (interface{}, error) {
func (s *Server) sendrawtransaction(reqParams Params) (interface{}, error) {
```
But I agree, it's not a good goesque.